### DISCUSSION TITLE
Why were field groups removed from the Craft 5?

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#14083] [Q&A] [answered]
This functionality was pretty useful for managing large amounts of fields.

**Comment by brandonkelly** [comment, marked as answer]
From what we’ve seen over the years, there are two common approaches to using field groups:

1. Create field groups that correlate to the section/entry type/global set/volume they’re used in
2. Complete chaos (either everything is in a single group, or the remnants of a strategy that was abandoned long ago)

Taking a more fussy approach gets awkward when you end up reusing a field in multiple field layouts, which became more common in Craft 3.5, when you could start overriding fields’ labels and instructions within the field layout. And we expect field reuse will explode in Craft 5 with the ability to create multiple instances of fields within a single layout.

So we decided it’s time to drop them. We think there are better ways to improve field management, which don’t involve rolling your own organizational system, such as a search bar (#13961), and the ability to create and edit fields inline within field layout managers (#8530).

**Reply by piotrpog** [reply]
@brandonkelly 
Thank you for explaining. It indeed does make sense considering the fact that we will be able to reuse the fields.
